settings/Screen_Height_216.txt

```diff
@@ -52,6 +52,7 @@ GreatGianaSisters
 HATE
 Harlequin1MB
 Heimdall
+IK+
```

Comment:
IK+ (as InternationalKarate+) is in the 240 height section?

I was only playing this a few days ago, I wish i could remember the value for it! I would like to just double-check for the sake of my sanity.

Reply:
IK+ is the new file/folder name. I noticed you added an entry for the old 'InternationalKarate+' in the Offset file. This didn't seem to be needed as Screen height 216 looks fine.

Reply:
Looks like conflicts are going to be a pain as the game name will probably need searching in all files !

Reply:
it would need to be in both, as some people may still be using the old file.

Reply:
Yes, but the same value !

Reply:
the behaviour for multiple instances is that it will search incremntally, 200,216,240,256 etc but stop searching once found. (i.e. lowest value is used)

Reply:
good example here, in that i have a small display where IK+ can fit into 200px with +8 vertical offset. I assume that is not the case for yourself?

Reply:
Yes I agree, Screen Height 200 and +8 Offset seems to fill more without missing anything. Could you change 'IK+' and 'InternationalKarate+' to this and delete the others ?